pcdsdevices/mirror.py

```diff
@@ -1206,6 +1206,14 @@ class FFMirrorZ(FFMirror):
     # RMS Cpts:
     z_enc_rms = Cpt(PytmcSignal, ':ENC:Z:RMS', io='i', kind='normal')
 
+    # RTDs
+    chin_left_rtd = Cpt(PytmcSignal, ':RTD:CHIN:L', io='i',
+                        kind='normal')
+    chin_right_rtd = Cpt(PytmcSignal, ':RTD:CHIN:L', io='i',
```

Comment:
(I know this is draft but I saw the diff and noticed the copy-paste error here: `right_rtd ... ':RTD:CHIN:L'`)